hail/python/hail/experimental/import_gtf.py

```diff
@@ -0,0 +1,132 @@
+
+import hail as hl
+from hail.utils import wrap_to_list
+
+
+def import_gtf(path, key=None):
+    """Import a GTF file.
+
+       The GTF file format is identical to the GFF version 2 file format,
+       and so this function can be used to import GFF version 2 files as
+       well.
+
+       See https://www.ensembl.org/info/website/upload/gff.html for more
+       details on the GTF/GFF2 file format.
+
+       The :class:`.Table` returned by this function will include the following
+       row fields:
+
+       .. code-block:: text
+
+           'seqname': str
+           'source': str
+           'feature': str
+           'start': int32
+           'end': int32
+           'score': float64
+           'strand': str
+           'frame': int32
+
+       There will also be corresponding fields for every tag found in the
+       attribute field of the GTF file.
+
+       .. note::
+
+           The "end" field in the table will be incremented by 1 in
+           comparison to the value found in the GTF file, as the end
+           coordinate in a GTF file is inclusive while the end
+           coordinate in Hail is exclusive.
```

Comment:
this isn't totally true -- intervals are parameterized by their start- and end-inclusivity. Could also leave end as it is and include a warning about using end-inclusive intervals? Up to you.

We could also construct the interval in this method, as well (given reference genome I guess?)